cmd20.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# Name: cmd20py
+# Authors: Luke Short | Joe R
+# Repo: https://github.com/ekultails/cmd20py/
+
+import random
+
+# Number Generator
+random.seed(None)
+die1 = random.randrange(0,7)
+die2 = random.randrange(0,7)
+die_results_msg = "You rolled a:"
```

Comment:
Lines 13 and 14 can be consolidated. Have a look at [PEP 3101](https://www.python.org/dev/peps/pep-3101/), this explains some advanced string substitutions you can do. For this example, you would want to do something along the lines of: `print("You rolled a: %d and %d" % (die1, die2))`. This will replace **d**igits with integer variables you pass to it. You can do similar things with strings by using %s instead.

Reply:
In PR https://github.com/ekultails/cmd20py/pull/6 , I ended up removing the print and just resorted to displaying the numbers until we are happy with the seed generation function. What do you think? Should we omit it for now or place it back in the code base?